package/MDAnalysis/analysis/MSD.py

```diff
@@ -128,22 +129,14 @@ class MSD(object):
 
     Methods
     -------
-    _select(self)
-        Generates list of atom groups that satisfy select_list strings
-    _init_pos_deque(self):
-        Initializes lists necessary for MSD calculations at the first restart.
-    _update_deques(self, this_restart):
-        Updates lists necessary for MSD calculations
-    _process_pos_data(self):
-        Runs MSD calculation
     run(self):
         Calls _process_pos_data and saves final output
 
     """
 
     def __init__(self, universe, select_list, start_frame, stop_frame,
-                 dt_restart, out_name='msd', len_msd=250, write_output=True,
-                 max_data=False):
+                 dt_restart, out_name='msd.csv', len_msd=250, write_output=True,
+                 max_data=False,verbose=False):
```

Comment:
In MDAnalysis we use `quiet` instead of `verbose`. Until we changed that (see #903) please use quiet